styles.css

```diff
@@ -426,6 +426,106 @@ textarea {
   border-radius: 10px;
 }
 
+/* modal styling */
+
+.popup-modal {
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-color: rgb(59.2%, 58%, 57.6%, 0.8);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  visibility: hidden;
+  backdrop-filter: blur(10px);
+}
+
+.popup-content {
+  width: 80%;
+  height: 80%;
+  background-color: white;
+  padding: 5%;
+}
```

Comment:
- [ ] kindly 😃 as stated by the previous reviewer, Good job implementing the changes and improving the UI of the project 👏 however, there are still some adjustments to be made. Currently, your popup does not align with the design. Kindly make some adjustments to your styling to implement these changes. Please check the image attached below for comparison and better understanding. 🙂

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Original design | Your design| 
| --- | --- |
|   ![image](https://github.com/mj-e-boc/M-J-Portfolio/assets/83360636/2a422d3b-1056-4007-91ac-16aaa058eb69)  |    ![image](https://github.com/mj-e-boc/M-J-Portfolio/assets/83360636/4c1fab5a-a9fe-450a-80ee-ab326d79fe9b)  |


> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Original design | Your design| 
| --- | --- |
|   ![image](https://github.com/mj-e-boc/M-J-Portfolio/assets/83360636/42a8793a-ba51-42d1-9205-dee9f8ab5209)   |   ![image](https://github.com/mj-e-boc/M-J-Portfolio/assets/83360636/fb3cdb9b-cca8-4ee5-8008-a91fd31128da)  |